sdk/spring/azure-spring-boot/src/main/resources/azure-spring-boot-version.txt

```diff
@@ -0,0 +1 @@
+version=${pom.version}
```

Comment:
In java world, it is common to use a ".properties" file for this kind of thing, instead of a "txt"

Reply:
Updated.